controllers/apply/form-router/lib/salesforce.js

```diff
@@ -30,7 +35,40 @@ class Salesforce {
             )}`
         ).then(function(result) {
             if (result.records.length > 0) {
-                return result.records[0]['Funding_Request__r']['Status__c'];
+                const rawStatus =
+                    result.records[0]['Funding_Request__r']['Status__c'];
+
+                let friendlyStatus = 'Unknown';
+                switch (rawStatus) {
```

Comment:
Added this based on the full list of pick-list values. I've been pretty arbitrary here so consider this a rough indication of what we might need to do, not real attempt.